massdriver.yaml

```diff
@@ -0,0 +1,69 @@
+schema: draft-07
+name: aws-sagemaker-inference-endpoint
+description: AWS SageMaker Inference Endpoint for hosting an AI model
+source_url: github.com/massdriver-cloud/aws-sagemaker-inference-endpoint
+access: public
+type: infrastructure
+
+params:
+  required:
+    - endpoint_config
+  properties:
+    endpoint_config:
+      title: Endpoint Configuration
+      type: object
+      required:
+        - primary_container
+        - instance_type
+        - instance_count
+      properties:
+        primary_container:
+          title: Primary Container
+          type: object
+          required:
+            - model_data
+            - ecr_image
+          properties:
+            model_data:
+              title: Model Data S3 URI
+              description: "The S3 URI of the model data. (e.g. s3://bucket-name/model.tar.gz)"
+              type: string
+            ecr_image:
```

Comment:
Can sagemaker host **any** model, or does it have to be one of "their" models? Wondering if we should use an enum here.

Reply:
It's any model. The goal is to allow people to host models they've finely tuned and model data they've therefore created. They'll do this in the Domain/Notebook Studio most likely and then save their model data in S3, then deploy it via Endpoint.